Dockerfile

```diff
@@ -1,5 +1,42 @@
-FROM ubuntu:18.04
+FROM amsterdam/python:3.8-buster as app
+MAINTAINER datapunt@amsterdam.nl
 
-EXPOSE 8089
-RUN apt-get update && apt-get install -y python3-pip
-RUN pip3 install locust
+ENV PYTHONUNBUFFERED 1
+ARG AUTHORIZATION_TOKEN
+ARG GET_AUTHORIZATION_TOKEN
+ARG SECRET_KEY
+
+EXPOSE 8000
+
+RUN mkdir -p /static && chown datapunt /static
```

Comment:
Volgens mij niet nodig

Reply:
Is wel nodig voor de static files die eronder worden geladen met collectstatic